source/tables.tex

```diff
@@ -457,7 +457,7 @@
 {
  \begin{LongTable}
  {#1}{#2}
- {x{.35\hsize}x{.28\hsize}x{.30\hsize}}
+ {x{.35\hsize}x{.28\hsize}x{.29\hsize}}
 }
```

Comment:
Interesting. I wonder were those original values came from. I suppose they were too big because they didn't account for cell padding?